python/cli/data_platform_workflows_cli/update_bundle.py

```diff
@@ -158,7 +158,7 @@ def fetch_postgresql_snaps(charm_revision) -> list[Snap]:
             result.append(Snap(
                 name=snap_name,
                 revision=int(snap_info["revision"]["x86_64"]),
-                push_channel=snap_info["channel"],
+                push_channel="14/stable",
```

Comment:
Now reconsidering the context of `push_channel`, I think it is indeed safer to use the edge channel, since it will always have whatever revision we're pinning. Will update accordingly!